e2e/quit-on-close.e2e.spec.ts

```diff
@@ -0,0 +1,59 @@
+import path from 'path';
+import util from 'util';
+
+import { test, expect } from '@playwright/test';
+import { ElectronApplication, _electron, Page } from 'playwright';
+
+import { createDefaultSettings, packageLogs, reportAsset } from './utils/TestUtils';
+
+const waitTime = 1000;
+const sleep = util.promisify(setTimeout);
+
+async function startRancherDesktop(): Promise<[ElectronApplication, Page]> {
+  const electronApp = await _electron.launch({
+    args: [
+      path.join(__dirname, '../'),
+      '--disable-gpu',
+      '--whitelisted-ips=',
+      // See pkg/rancher-desktop/utils/commandLine.ts before changing the next item as the final option.
+      '--disable-dev-shm-usage',
+      '--no-modal-dialogs',
+    ],
+    env: {
+      ...process.env,
+      RD_LOGS_DIR: reportAsset(__filename, 'log'),
+    },
+  });
+  const page = await electronApp.firstWindow();
+  return [electronApp, page];
+}
+
+/**
+ * Using test.describe.serial make the test execute step by step, as described on each `test()` order
+ * Playwright executes test in parallel by default and it will not work for our app backend loading process.
+ * */
+test.describe.serial('quitOnClose setting', () => {
+  test.afterEach(async() => {
+    await packageLogs(__filename);
```

Comment:
Packaging the logs after _each_ test means that it'll clobber the previous test logs (from the same run), right? Should this be `afterAll` instead?

It may be useful to use the [teardown helper](https://github.com/rancher-sandbox/rancher-desktop/blob/main/e2e/utils/TestUtils.ts#L81), but that probably depends on the answer to the previous question…